pydis_site/apps/api/tests/test_reminders.py

```diff
@@ -46,35 +46,36 @@ def test_delete_returns_404(self):
 
 
 class ReminderCreationTests(APISubdomainTestCase):
-    def setUp(self):
-        super().setUp()
-        self.author = User.objects.create(
+    @classmethod
+    def setUpTestData(cls):
+        cls.author = User.objects.create(
             id=1234,
             name='Mermaid Man',
             discriminator=1234,
             avatar_hash=None,
         )
-        self.data = {
+
+    def test_accepts_valid_data(self):
+        data = {
             'author': self.author.id,
             'content': 'Remember to...wait what was it again?',
             'expiration': datetime.utcnow().isoformat(),
             'jump_url': "https://www.google.com",
             'channel_id': 123,
         }
         url = reverse('bot:reminder-list', host='api')
-        response = self.client.post(url, data=self.data)
+        response = self.client.post(url, data=data)
         self.assertEqual(response.status_code, 201)
+        self.assertIsNotNone(Reminder.objects.get(id=1))
```

Comment:
My concern is `get()` raises an error rather than returning `None`. A test failure due to an uncaught exception may be less clear than an assertion failing.

Reply:
Ah, I didn't consider that, thanks. I did wonder why your initial suggestion used `filter(...).first()`, and now I get it.